site/js/app.js

```diff
@@ -752,7 +753,12 @@ var pin_remove_button = function(opt_options) {
   });
 };
 
-ol.inherits(pin_remove_button, ol.control.Control);
+var ol_ext_inherits = function(child,parent) {
+    child.prototype = Object.create(parent.prototype);
+    child.prototype.constructor = child;
+};
+
+ol_ext_inherits(pin_remove_button, ol.control.Control);
 
```

Comment:
I am not too happy about defining a function and then just using it once. I realize that this is really just a workaround for getting this to run again with minimal effort and there are probably better ways to do this anyway. But for the time being I suggest doing this directly without the function.

Reply:
I have started to rewrite some code using the newer Javascript `class` syntax, then the whole `inherits` stuff goes away. So never mind my comment, just leave the `ol_ext_inherits` as it is for now and we'll get rid of it soon anyway.